package.json

```diff
@@ -71,5 +70,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "workspaces": [
```

Comment:
Opted to somewhat simplify the setup by just including `PublishHtmlReport` as a workspace, instead of having two separate projects and using the `postinstall` task. Can definitely undo this behavior.